lib/FFI/Platypus/Memory.pm

```diff
@@ -103,9 +103,33 @@ useful in Perl.
 The same as C<strdup> above, except at most C<$max> characters will be
 copied in the new string.
 
+=head2 strlenW
+
+ my $size = strlenW $opaque;
+
+Counts the length of a null-terminated string of 16-bit characters.
+This is useful when dealing with Windows Unicode strings (LPCWSTR and LPWSTR)
+and other UTF-16 libraries.
+
+=head2 strcpyW
+
+ my $size = strcpyW $opaque, $opaque;
+
+The same as C<strcpy> above, except it works with 16-bit characters.
+This is useful when dealing with Windows Unicode strings (LPCWSTR and LPWSTR)
+and other UTF-16 libraries.
+
+=head2 strncpyW
+
+ my $size = strncpyW $opaque, $opaque, $max;
+
+The same as C<strncpy> above, except it works with 16-bit characters.
+This is useful when dealing with Windows Unicode strings (LPCWSTR and LPWSTR)
+and other UTF-16 libraries.
+
 =cut
 
-our @EXPORT = qw( malloc free calloc realloc memcpy memset strdup strndup strcpy );
+our @EXPORT = qw( malloc free calloc realloc memcpy memset strdup strndup strcpy strlenW strcpyW strncpyW );
```

Comment:
I'm conflicted on whether these should be `@EXPORT` or  `@EXPORT_OK`.

Reply:
I never use `@EXPORT`, so I just followed suit. This becomes moot if they're moved to a separate module.

Reply:
We will keep it `@EXPORT` here, an explicit empty import is what I usually use anyway.